Jenkinsfile

```diff
@@ -1,55 +1,28 @@
-def formElementPathVersion
-node('hi-speed') {
-    stage('plugin') {
-        dir('plugin') {
-            checkout scm
-            timeout(30) {
-                withMaven(jdk: 'Oracle JDK 8 (latest)',
-                        maven: 'Maven 3.2.1') {
-                    sh 'mvn clean install -B'
-                    formElementPathVersion = sh script: 'cat target/form-element-path/META-INF/MANIFEST.MF | grep Implementation-Version | cut -f 2 -d ":" | xargs', returnStdout: true
-                }
+def mavenName = 'mvn'
+def jdkName = 'jdk7'
+
+node('highram&&docker') {
+    stage 'plugin'
+    dir('plugin') {
+        checkout scm
+        timeout(30) {
+            withMaven(jdk: jdkName, maven: mavenName) {
+                sh 'mvn clean install -B'
             }
         }
     }
+    stage 'Acceptance Tests'
     dir('ath') {
-        stage('Acceptance Tests') {
-            git changelog: false, poll: false, url: 'git@github.com:jenkinsci/acceptance-test-harness.git'
-            step([$class: 'CopyArtifact',
-                  filter: 'war/target/jenkins.war',
-                  fingerprintArtifacts: true,
-                  flatten: true,
-                  projectName: 'core/jenkins_lts_branch',
-                  selector: [
-                          $class: 'StatusBuildSelector',
-                          stable: false
-                  ]
-            ])
-            try {
-                wrap([$class: 'Xvnc', takeScreenshot: false, useXauthority: true]) {
-                    withMaven(jdk: 'Oracle JDK 8 (latest)',
-                            maven: 'Maven 3.2.1') {
-                        withEnv(['BROWSER=firefox',
-                                 'JENKINS_WAR=jenkins.war',
-                                 'JENKINS_JAVA_HOME=/opt/jdk/jdk1.7.latest/',
-                                 'JAVA_HOME=/opt/jdk/openjdk8.latest/',
-                                 'EXERCISEDPLUGINREPORTER=textfile',
-                                 "FORM_ELEMENT_PATH_VERSION=${formElementPathVersion.trim()}"]) {
-                            sh '''
-unzip -p $JENKINS_WAR META-INF/MANIFEST.MF | perl -p -0777 -e 's/\\r?\\n //g' | fgrep Jenkins-Version
-cp $SAUCE_SECRET ~/.sauce-ondemand || true
-
-firefox --version
-mvn clean test -Dmaven.test.failure.ignore=true -DforkCount=1 -B
-'''
-                        }
-                    }
+        git changelog: false, poll: false, url: 'https://github.com/jenkinsci/acceptance-test-harness.git'
+        sh 'wget http://mirrors.jenkins.io/war-stable/latest/jenkins.war -O jenkins.war'
+        try {
+            withMaven(jdk: jdkName, maven: mavenName) {
+                withEnv(['BROWSER=firefox', 'JENKINS_WAR=jenkins.war']) {
```

Comment:
Out of curiosity, how will these tests be launching Firefox (or will they at all)? There is no open X11 servers available (as one might expect) so the test suite is hopefully performing that work.